Jose/Jwt.hs

```diff
@@ -145,4 +149,26 @@ decodeClaims jwt = do
     claims <- B64.decode ((head . tail) components) >>= parseClaims
     return (hdr, claims)
   where
-    parseClaims bs = maybe (Left BadClaims) Right $ decodeStrict' bs
+    parseClaims bs =
+      maybe
+      ( Left
+      $ BadClaims
+        "Failed to decode claims. For more details, use `eitherDecodeClaims`."
```

Comment:
If we are going to add an error message to `BadClaims` then I guess that is a breaking change for existing code which pattern matches on the error, so I'd prefer to just modify the existing decode function here to record the error, not bother with the extra `eitherDecodeClaims` version (which has the same signature) and just bump the library version to 1.0.